src/components/Title/Title.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface TitleProps {
+  className?: string
```

Comment:
this is _extremely_ nitpicky but `className` is part of `React.HTMLAttributes` and you can therefore omit it here